modules/parser/chromomap_parser.nf

```diff
@@ -0,0 +1,18 @@
+process chromomap_parser {
+    publishDir "${params.output}/${name}/hmm/chromomap_filter", mode: 'copy'
+    label 'noDocker'
+
+    input:
+    tuple val(name), file(positive_contigs_list), file(hmmscan_results), file(prodigal_out)
+    file(vogtable)
```

Comment:
why is this file not part of the channel above?

Reply:
try `.combine()`  instead of join for this case

Reply:
* but if it works stable than its okay (resolve this conversation then)

Reply:
it works stable. I will have a look when i work on #59